src/TraceEvent/Parsers/ClrTraceEventParser.cs

```diff
@@ -9882,4 +9950,85 @@ public enum GCGlobalMechanisms
         Demotion = 0x8,
         CardBundles = 0x10,
     }
+
+    #region private types
+    /// <summary>
+    /// ClrTraceEventParserState holds all information that is shared among all events that is
+    /// needed to decode Clr events.   This class is registered with the source so that it will be
+    /// persisted.  Things in here include
+    /// 
+    ///     * TypeID to TypeName mapping, 
+    /// </summary>
+    // [SecuritySafeCritical]
+    internal class ClrTraceEventParserState : IFastSerializable
+    {
+        internal void SetTypeIDToName(int processID, Address typeId, long timeQPC, string typeName)
```

Comment:
💭 If you can't foresee a case where we would likely want to use a non-zero value going in, it seems like removing the parameter and commenting in the body of the method makes it less likely to misuse this. Otherwise the knowledge requirement is pushed onto the caller. But it's internal and probably rarely updated, so minor.